host/xtest/regression_1000.c

```diff
@@ -1160,10 +1160,10 @@ static void xtest_tee_test_1014(ADBG_Case_t *c)
 	Do_ADBG_EndSubCase(c, "SDP: SDP TA invokes a SDP pTA");
 
 	test = TEST_NS_TO_PTA;
-	Do_ADBG_BeginSubCase(c, "SDP: NonSecure client invokes SDP pTA (should fail)");
+	Do_ADBG_BeginSubCase(c, "SDP: CA invokes SDP pTA (should fail)");
```

Comment:
Can't a TA be a CA?

Reply:
Yes, true. I will keep the `NSec` attribute.